Tests/FirebaseTests/Tests/DynamicLinks/DynamicLinksRoutesTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+@testable import Firebase
+@testable import Vapor
+
+final class DynamicLinksRoutesTests: XCTestCase {
+
+    func test_routeRequests_should_beInJson_when_encodedBody() throws {
+        // Given
+        let request = DynamicLinkRequest(dynamicLink: DynamicLinkInfo(link: "http://link"),
+                suffix: Suffix(option: .SHORT))
+        let expectedJson = "{\"suffix\":{\"option\":\"SHORT\"},\"dynamicLinkInfo\":{\"link\":\"http:\\/\\/link\"}}"
+
+        // When
+        let json = try request.toEncodedBody()
+
+        // Then
+        XCTAssertEqual(json, expectedJson)
+    }
+
+    func test_createShortLink_should_callShortLinksEndpointOnceTime_on_call() throws {
```

Comment:
Naming should be `test_createShortLink_should_callShortLinksEndpointOnce_on_call()` or `test_createShortLink_should_callShortLinksEndpointOneTime_on_call()`

Reply:
https://github.com/h2glab/firebase-provider/pull/3/commits/d6d756dc72192a0d4d003929589046a443df4802